Remove flash based xtypes

Jako opened the pull request:
### What does it do?
Remove flash based xtypes

### Why is it needed?
It could be possible, that sites running any Flash will be actively blocked by Chrome.

### How to test
The manager works with this changes and the xtypes removed are not used in the core.

### Related issue(s)/PR(s)
https://github.com/modxcms/revolution/issues/13199#issuecomment-717908136

Jako committed:
Remove flash based xtypes
'chart', 'piechart', 'cartesianchart' and the derived xtypes

Jako commented:
This PR could be backported/copied to 2.x

JoshuaLuckers commented:
What if an extra relies on these?

Jako commented:
Then the extras have a different issue, because browser based flash is EOL. 

Maybe we could add the removed parts to an extra file.

Jako committed:
Add the chart components in an extra file.

Ruslan-Aleev commented:
Maybe in the future it makes sense to transfer/remove other components that are not used?
If there are many of them, then by removing them we will lighten the size of the distribution, we can avoid potential errors in the future (with flash, for example) and even update extJS components separately.

Jako committed:
Deactivate MODx.util.Clipboard

Jako commented:
http://swfobject.googlecode.com/svn/trunk/swfobject/expressInstall.swf and http://yui.yahooapis.com/2.8.2/build/charts/assets/charts.swf don't exist anymore. So the Ext.chart things could be removed. They can't be used anymore and should currently throw an error.

Jako committed:
Remove the chart components finally

Jako commented:
The FileApi code has a fallback to flash, but the referenced files are not available anymore. It could be disabled quite easy.

Jako committed:
Disable flash fallback

JoshuaLuckers commented on manager/assets/modext/util/utilities.js:
```diff
@@ -421,28 +421,8 @@ Ext.applyIf(Ext.form.Field,{
-                document.getElementById(flashcopier).innerHTML = divinfo;
-            }
+        copy: function(text){
+            console.log('MODx.util.Clipboard was deactivated!');
```
I suggest removing this component completely. If there are good reasons to keep it we should log an error instead.

Jako replied:
A github code search for MODx.util.Clipboard shows 459 results, most in the utilities.js file or in some cache files. So I think that the component is no longer in use.

Jako committed:
Completely remove MODx.util.Clipboard

A github search for MODx.util.Clipboard filename:utilities.js shows 459 results, most in the utilities.js file or in some cache files. So I think that the component is no longer in use.

opengeek committed:
grunt build
